[2023-10-17T11:19:15.142+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-17T11:19:15.151+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-17T11:19:15.152+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:15.162+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-23 00:00:00+00:00
[2023-10-17T11:19:15.171+0000] {standard_task_runner.py:57} INFO - Started process 597 to run task
[2023-10-17T11:19:15.173+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-23T00:00:00+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmptul42tgp']
[2023-10-17T11:19:15.174+0000] {standard_task_runner.py:85} INFO - Job 86: Subtask waiting_for_data
[2023-10-17T11:19:15.226+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:15.327+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-23T00:00:00+00:00'
[2023-10-17T11:19:15.338+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:15.338+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:15.339+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:15.339+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:15.347+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210323T000000, start_date=20231017T111915, end_date=20231017T111915
[2023-10-17T11:19:15.388+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:15.408+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:44.850+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-18T10:42:44.860+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-18T10:42:44.861+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:44.879+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-23 00:00:00+00:00
[2023-10-18T10:42:44.885+0000] {standard_task_runner.py:57} INFO - Started process 1561 to run task
[2023-10-18T10:42:44.888+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-23T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmphmff0itu']
[2023-10-18T10:42:44.890+0000] {standard_task_runner.py:85} INFO - Job 85: Subtask waiting_for_data
[2023-10-18T10:42:44.936+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:45.008+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-23T00:00:00+00:00'
[2023-10-18T10:42:45.026+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:45.030+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210323T000000, start_date=20231018T104244, end_date=20231018T104245
[2023-10-18T10:42:45.044+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 85 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1561)
[2023-10-18T10:42:45.060+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:45.091+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:18.977+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-18T11:17:18.986+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-18T11:17:18.987+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:19.000+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-23 00:00:00+00:00
[2023-10-18T11:17:19.006+0000] {standard_task_runner.py:57} INFO - Started process 387 to run task
[2023-10-18T11:17:19.008+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-23T00:00:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp5_6p3pho']
[2023-10-18T11:17:19.009+0000] {standard_task_runner.py:85} INFO - Job 128: Subtask waiting_for_data
[2023-10-18T11:17:19.043+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:19.119+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-23T00:00:00+00:00'
[2023-10-18T11:17:19.129+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:19.130+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:19.131+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:19.131+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:19.143+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210323T000000, start_date=20231018T111718, end_date=20231018T111719
[2023-10-18T11:17:19.180+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:19.249+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:56.055+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-19T12:11:56.065+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [queued]>
[2023-10-19T12:11:56.065+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:56.122+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-23 00:00:00+00:00
[2023-10-19T12:11:56.129+0000] {standard_task_runner.py:57} INFO - Started process 433 to run task
[2023-10-19T12:11:56.132+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-23T00:00:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpls0s65wl']
[2023-10-19T12:11:56.133+0000] {standard_task_runner.py:85} INFO - Job 124: Subtask waiting_for_data
[2023-10-19T12:11:56.192+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-23T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:56.275+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-23T00:00:00+00:00'
[2023-10-19T12:11:56.285+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:56.286+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:56.286+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:56.286+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:56.294+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210323T000000, start_date=20231019T121156, end_date=20231019T121156
[2023-10-19T12:11:56.349+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:56.384+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
